apps/educational_institutions/handlers.py

```diff
@@ -33,27 +41,31 @@ async def create_faculty(
         if not data.dean_id:
             dean = await self.create_dean(
                 request=request,
-                data={
-                    "last_name": data.dean_last_name,
-                    "first_name": data.dean_first_name,
-                    "middle_name": data.dean_middle_name,
-                },
+                data=FullNameSchema(
+                    last_name=data.dean_last_name,
+                    first_name=data.dean_first_name,
+                    middle_name=data.dean_middle_name,
```

Comment:
did u test it with empty middle_name value or without key "middle_name"?

Reply:
Of course tested it. Middle_name must be empty (null as value) or filled with correct data. And the "Update error" problem is present, fixed in 81 ticket by deleting id values from inserting info.

Reply:
ok, i'll test myself today to be sure and if it's work correct - i'll merge it to the release branch

Reply:
tested, works good